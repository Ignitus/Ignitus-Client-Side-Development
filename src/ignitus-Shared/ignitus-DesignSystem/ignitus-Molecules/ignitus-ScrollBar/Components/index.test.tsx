import React from 'react';
import { render } from 'enzyme';
import emotionSerializer from 'jest-emotion';
import { ScrollBar } from './index';

expect.addSnapshotSerializer(emotionSerializer);

describe('<ScrollBar />', () => {
  it('should render', () => {
    const wrapper = render(<ScrollBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
