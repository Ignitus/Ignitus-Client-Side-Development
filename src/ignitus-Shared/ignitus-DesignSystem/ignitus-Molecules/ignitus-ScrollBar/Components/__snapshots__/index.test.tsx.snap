// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScrollBar /> should render 1`] = `
.emotion-1 {
  margin: 30px;
  float: left;
  height: 300px;
  width: 65px;
  background: #f5f5f5;
  overflow-y: scroll;
  margin-bottom: 25px;
}

.emotion-1::-webkit-scrollbar {
  width: 12px;
  background: #f5f5f5;
}

.emotion-1::-webkit-scrollbar-track {
  box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
  border-radius: 10px;
  background-color: #f5f5f5;
}

.emotion-1::-webkit-scrollbar-thumb {
  border-radius: 10px;
  background-color: #f90;
}

.emotion-0 {
  min-height: 450px;
}

<div
  class="emotion-1"
>
  <div
    class="emotion-0"
  />
</div>
`;
